pbsmetrics/prometheus/prometheus.go

```diff
@@ -73,11 +74,17 @@ const (
 	markupDeliveryNurl = "nurl"
 )
 
+const (
+	requestSuccessLabel = "accepted"
```

Comment:
Consider renaming to `requestAcceptedLabel` to better match its value.

Reply:
Refactored